bookshelf-standard/2-structured-data/pom.xml

```diff
@@ -134,10 +134,6 @@ Copyright 2016 Google Inc.
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-server</artifactId>
     </dependency>
-    <dependency>
```

Comment:
Yeah - I realized that the dependency wasn't actually necessary :-) Apparently it's only necessary if you want to do chrome- or firefox- specific stuff, but I'm only using the generic RemoteWebDriver (which for all I know includes this already as a dependency. Anyway, it works without this dep, so I figured I'd clean up)